sadl3/com.ge.research.sadl.parent/com.ge.research.sadl.jena/src/com/ge/research/sadl/jena/JenaBasedSadlModelProcessor.java

```diff
@@ -10726,7 +10726,16 @@ private EnumeratedClass sadlExplicitValuesToEnumeratedClass(EList<SadlExplicitVa
 		List<RDFNode> nodevals = new ArrayList<RDFNode>();
 		for (int i = 0; i < values.size(); i++) {
 			SadlExplicitValue value = values.get(i);
-			RDFNode nodeval = sadlExplicitValueToRdfNode(value, null, true);
+			if (value instanceof SadlResource) {
+				try {
+					OntConceptType vtype = getDeclarationExtensions().getOntConceptType((SadlResource)value);
+					if (!vtype.equals(OntConceptType.INSTANCE)) {
+						addError("Expected an instance in the enumeration of the class", value);
+					}
+				} catch (CircularDefinitionException e) {
+					throw new JenaProcessorException("Unexpected error, please report: " + e.getMessage());
+				}
+			}			RDFNode nodeval = sadlExplicitValueToRdfNode(value, null, true);
```

Comment:
Line 10738 looks funny because there is both a closing `}` and a statement on the same line.  Normally a closing `}` would be on a line by itself.  Will you please move the statement to its own separate line after the closing `}`?

Reply:
Ah, didn't notice it on paste.